utils/file.js

```diff
@@ -36,6 +36,25 @@ module.exports = {
     await smartWait(isFileCreated, [filePath], timeout, errorMessage);
   },
 
+  /**
+   * Create a file in local storage
+   */
+  async createTmpFileWithRandomeNameAndContent() {
+    let rand = (Math.random() + 1).toString(36).substring(2,7); // 5 random chars
+    const fileName = `qa-upload-file_${rand}.txt`;
+    const filePath = './' + fileName;
+    const fileContents = 'this fake data file was generated and uploaded by the integration test suite\n';
```

Comment:
the backend data upload tests need access to `fileContents` to check the contents of the file after downloading it. what do you think about naming this function `createTmpFileWithRandomName(fileContents)`?